pubsubtests/src/main/java/id/pubsubtests/PubSubClientTests.java

```diff
@@ -209,7 +205,7 @@ public void test_publisher_on_close(TestCase testCase) throws Exception {
             subscriberClient.subscribe(topic, collector);
             publisher.submit(data);
             // sleep until subscriber is discovered by publisher
-            while (received.isEmpty()) XThread.sleep(100);
```

Comment:
Hi @dbortoluzzi,

Thank you for the MR it looks great.
I have one question about this while loop, not sure why it is removed?